From: Julian Wollrath <jwollrath@web.de>
Date: Sat, 23 Nov 2013 13:45:41 +0100
Subject: Do not generate css.h with csstoh.sh

---
 src/Makefile.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/Makefile.in b/src/Makefile.in
index 7d6858622ea8..35235be83e86 100644
--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -1873,9 +1873,6 @@ uninstall-am: uninstall-sbinPROGRAMS
 	mostlyclean-generic mostlyclean-libtool pdf pdf-am ps ps-am \
 	tags tags-am uninstall uninstall-am uninstall-sbinPROGRAMS
 
-css.h: powertop.css
-	$(SHELL) ./csstoh.sh powertop.css css.h
-
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
